### DISCUSSION TITLE
Setting up Airflow in AWS Linux Machine

### DISCUSSION THREAD

**Discussion by vijaya-lakshmi-venkatraman** [apache/airflow#22425] [Q&A]
Hello,
I am trying to set up Airflow in an AWS Linux machine (with browsing proxy for internet access).
python version = 3.7.10 & pip v22.0.4 (from python3.7).
I have set up proxy for git, pip as well as http_proxy & https_proxy in bash_profile.rc file.
I'm stuck at step 15/36 where it stops at install_pipx_tools.sh due to _**ConnectTimeoutError**_, stating that connection to pypi.org timed out.
I tried installing pipx separately and it is successfully installed (v1.0.0) but still the error persists.

Is there a way to get past this error?

**Comment by potiuk** [comment]
Thats something wrong in your system. You need to fix the networking to allow you to connect directly to PyPI for Breeze/development image. You will have mutliple other problems if you do not have it.
